api/namex/models/request.py

```diff
@@ -326,6 +326,23 @@ def find_name_by_choice(cls, nr_id, choice):
         name_by_choice = next((name for name in names if name.choice == choice), None)
         return name_by_choice
 
+    @classmethod
+    def find_existing_name_by_user(cls, user_data, request_owner):
+        """	
+        Gets requests submited by user with given name choice in state draft
+        """
+        existing_nr = db.session.query(Request). \
+            join(Applicant, and_(Applicant.nrId == Request.id)). \
+            filter(
+                Applicant.emailAddress == request_owner,
+                Request.stateCd == 'DRAFT',
+                Request.nameSearch.ilike('%'+user_data+'%')). \
```

Comment:
we only consider 'duplicate' name in the ticket and it means exact the same name. Please change it to "=" instead of "like"